projects/Dispersion/Dispersion.cpp

```diff
@@ -216,18 +216,15 @@ namespace projects {
       creal dy = cellParams[CellParams::DY];
       creal z = cellParams[CellParams::ZCRD];
       creal dz = cellParams[CellParams::DZ];
-      
-      CellID cellID = (int) ((x - Parameters::xmin) / dx) +
-         (int) ((y - Parameters::ymin) / dy) * Parameters::xcells_ini +
-         (int) ((z - Parameters::zmin) / dz) * Parameters::xcells_ini * Parameters::ycells_ini;
 
-      setRandomSeed(cellID);
+      std::default_random_engine rndState;
```

Comment:
What is this thing? Is this going to guarantee the same random result for a given cell no matter what the task+thread layout is? (Comment valid for all further occurrences.)

Reply:
OK. Good I guess then?